docs/user/ensure_your_experiments_are_reproducible.md

```diff
@@ -0,0 +1,46 @@
+# Ensure your experiments are reproducible
+Ensure the reproducibility of your experiments with comprehensive launcher files. Launcher files are used to initialize the an algorithm, its components (policy, replay buffer, etc.), and tools for managing experiments.
+
+There are some important take-aways from this example launcher file.
+
+```python
+from garage import wrap_experiment
+from garage.envs import GarageEnv
+from garage.experiment import deterministic, LocalRunner
+import a.b.c
+
+@wrap_experiment(wrap_experiment_args, wrap_experiment_kwargs)
+def experiment(ctxt=None, seed):
+    deterministic.set_seed(seed)
+    runner = LocalRunner(snapshot_config=ctxt)
+    env = GarageEnv(YOUR_ENV)
+
+    # Initialize algorithm dependencies (policy, Q-functions,
+    # replay buffers, etc.)
+    policy = ...
+    algo = ...
+
+    runner.setup(algo=algo, env=env)
+    runner.train(n_epochs=num_epochs, batch_size=)
+ 
+seed = 0
+experiment(seed) 
+```
+
+## Important Take Aways
+
+### Use `garage.wrap_experiment` to enable snapshotting and to save your results to a single directory
```

Comment:
perhaps you can mention the feature which stores the launcher directory?

Reply:
ofc yes, I'll include a description on this